Document CSSFontFeatureValuesRule (#25570)
mdn/content: files/en-us/web/api/cssfontfeaturevaluesrule/index.md

---
title: CSSFontFeatureValuesRule
slug: Web/API/CSSFontFeatureValuesRule
page-type: web-api-interface
browser-compat: api.CSSFontFeatureValuesRule
---

{{APIRef("CSSOM")}}

> The **`CSSFontFeatureValuesRule`** interface represents an {{cssxref("@font-feature-values")}} [at-rule](/en-US/docs/Web/CSS/At-rule), letting developpers assign for each font face a common name to specify features indices to be used in {{cssxref("font-variant-alternates")}}.

Reviewer: ```suggestion
The **`CSSFontFeatureValuesRule`** interface represents an {{cssxref("@font-feature-values")}} [at-rule](/en-US/docs/Web/CSS/At-rule), letting developers assign for each font face a common name to specify features indices to be used in {{cssxref("font-variant-alternates")}}.
```

{{InheritanceDiagram}}

## Instance properties

_Inherits properties from its ancestor {{domxref("CSSRule")}}._

- {{domxref("CSSFontFeatureValuesRule.fontFamily")}}

>   - : A string identify the font-family this rule applies to.

Reviewer 2: ```suggestion
  - : A string that identifies the font-family this rule applies to.
```

>   - : A string that identifies the font-family this rule applies to.

Reviewer: ```suggestion
  - : A string that identifies the font family this rule applies to.
```

## Instance methods

_Inherits methods from its ancestor {{domxref("CSSRule")}}._

## Examples

### Read font family

> In this example, we declared to {{cssxref("@font-feature-values")}} one fo the _Font One_ font family, and the other for _Font Two_. We then uses the CSSOm to read these font families, displaying them into the log.

Reviewer 2: ```suggestion
In this example, we declared two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```

> In this example, we declared two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.

Reviewer: ```suggestion
In this example, we declare two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```
